call-notes/call_7.md

```diff
@@ -0,0 +1,213 @@
+
+We would also add a RPC method that would allow anyone to just call that method and get a proof for the particular transaction.
+
+### Why Solana Need Light Clients
+
+Video | [03:33](https://youtu.be/9m_M8zEw1cE?t=213)
+-|-
+
+***Anoushk | Tinydancer:** So, as I mentioned, the reason that we need this is because you know RPCs could give you incorrect information that the transaction actually succeeded or it was included in the block but it actually didn't and that's an attack vector. You can still verify this data if you have a snapshot, but obviously snapshots are you know 30, 40 Gigabytes and that is a lot of data for the end user. Also, relating to this SIMD is about staking attestations. You can actually use transaction proofs to verify that a particular validator has a certain stake off chain. As mentioned in the SPV proposal a validator could technically use transaction proofs to act like a checkpoint to verify a certain state. And as part of a different proposal that uses SPV, it can also be used for inter-chain verification.
+
+Here is a list of important resources that you could take a look at this SIMD that we wrote SPV proposal. There is an open issue that talks about adding statuses to the [bank hash](https://docs.solana.com/proposals/simple-payment-and-state-verification#block-headers) and that is the inter-chain and SPV proposal. So, this is the part that there has been a lot of discussion on. And we think that there need to be more discussions on different parts of the code from community on like different ways to implement it.
+
+### Implementation (Method 1)
+
+Video | [05:30](https://youtu.be/9m_M8zEw1cE?t=330)
+-|-
+
+**Anoushk | Tinydancer:**  In our SIMD right now we are really implementing the first way which is modifying the block hash to include statuses but over the last week after discussing with some members of the [Jump team](https://jumpcrypto.com/firedancer/) and other members of Solana Labs. We also found different ways to implement it to tackle different issues that may arise while implementing the first method, that is modifying the block hash. So I am going to dive into each of these very briefly.
+
+So, modifying the block hash is pretty straightforward and again part of the history proposal so all it does is you add the transaction status along with the transaction signature into the merkle tree of the transactions which is part of each entry. And that gets hashed, gets mobilized into the block hash and becomes part of the bank hash. Currently, the block hash is a sequential hash of all the entries but making it would be better in terms of verification size and obviously having the status is important for verifying if transaction actually succeeded.
```

Comment:
```suggestion
So, modifying the block hash is pretty straightforward and again part of the proposal so all it does is you add the transaction status along with the transaction signature into the merkle tree of the transactions which is part of each entry. And that gets hashed, gets mobilized into the block hash and becomes part of the bank hash. Currently, the block hash is a sequential hash of all the entries but making it would be better in terms of verification size and obviously having the status is important for verifying if transaction actually succeeded.
```